netlify.toml

```
@@ -195,5 +195,6 @@ redirects = [
 
   { from = "/youtube", to = "https://www.youtube.com/@wasmCloud", status = 301 },
   { from = "/yt", to = "https://www.youtube.com/@wasmCloud", status = 301 },
+  { from = "/innovation-day", to = "https://www.youtube.com/watch?v=vEHbZCBmPok", status = 301 },
```

```suggestion
  { from = "/innovation-day-2024", to = "https://www.youtube.com/watch?v=vEHbZCBmPok", status = 301 },
```

Should we add a year? We might do this again

---

Yeah, I like the both years approach!